### DISCUSSION TITLE
能否支持用new bing翻译

### DISCUSSION THREAD

**Discussion by seeflood** [immersive-translate/immersive-translate#459] [Ideas]
### 新功能 | New Feature

插件支持使用new bing翻译

### 为什么您想要添加这些功能 | Why you want to add these features

目前只有少数人能使用gpt-4 api，如果能为插件添加“使用new bing 翻译”的功能，可以让大家免费享受到gpt-4的能力，而且对于国内用户new bing注册起来比openai 方便（不需要折腾信用卡）

### 提议 | Proposal

方案1. 需要用户先打开 new bing页面，插件发消息给 new bing页面、自动提问、自动获取页面上的返回结果。不确定反爬难度有多大

方案2. 逆向new bing 的api，直接调api。有可能被封号

**Comment by versun** [comment]
new bing有条数限制的哈,不大适合做为翻译

**Reply by seeflood** [reply]
emmmm 条数限制可以重置(新开一个页面就能重置了）。
我最近看论文一直用的new bing翻译，翻译效果比别的好……
<img width="206" alt="image" src="https://github.com/immersive-translate/immersive-translate/assets/26001097/9130017c-5fc2-419e-a523-5f5d33244c75">

**Comment by srsng** [comment]
条数限制是指每天限量，正常一天不够用，而且响应速度也是个问题。
